docs/FAQ.md

```diff
@@ -95,7 +95,7 @@ which may require multiple invocations of `jj rebase -r` or `jj rebase -s`.
 
 To squash or split commits, use `jj squash` and `jj split`.
 
-### How can I keep my scratch files in the repository?
+### How can I create scratch files without comitting them to the repository?
```

Comment:
Aside: This is not strictly related to this PR, but there is another trick for this I found. We're actually using it to not track `.jj`. I found it useful to not track `.vscode` in the `jj` repo without ignoring it in all repos (some repos put important workflow config in .vscode). 

You can create any dir with .gitignore containing `*` and it won't be tracked.

Or, if you want to be really surgical, you can probably put a `.gitignore` in any of the existing dirs that don't already have one and put

```
.gitignore
my_scratch_file
```

in it.

Reply:
I added a paragraph about this, thanks.